File: android/app/src/main/kotlin/com/example/flutter_seringueiro/PontoDeSangria.kt
package com.example.flutter_seringueiro

data class PontoDeSangria(
    val id: Int,
    val timestamp: Long,
    val latitude: Double,
    val longitude: Double,
    val duracao: Int,
    val sangriaId: String
) {
    fun toMap(): Map<String, Any> {
        return mapOf(
            "id" to id,
            "timestamp" to timestamp,
            "latitude" to latitude,
            "longitude" to longitude,
            "duracao" to duracao,
            "sangriaId" to sangriaId
        )
    }
}

File: lib/views/main/home/property/sangria/ponto_de_sangria.dart
class PontoDeSangria {
  final int id;
  final int timestamp;
  final double latitude;
  final double longitude;
  final int duracao;
  final String sangriaId;

  PontoDeSangria({
    required this.id,
    required this.timestamp,
    required this.latitude,
    required this.longitude,
    required this.duracao,
    required this.sangriaId,
  });

  Map<String, dynamic> toMap() {
    return {
      'id': id,
      'timestamp': timestamp,
      'latitude': latitude,
      'longitude': longitude,
      'duracao': duracao,
      'sangriaId': sangriaId,
    };
  }
}
